index.js

```diff
@@ -1,14 +1,16 @@
 'use strict';
-module.exports = function (a, b) {
+module.exports = function (a, b, minComp) {
 	if (!Buffer.isBuffer(a) || !Buffer.isBuffer(b)) {
 		throw new TypeError('Arguments must be Buffers');
 	}
 
+	var len = Math.max(a.length, b.length, minComp || 0);
```

Comment:
The problem is, that implementers can't enlarge the buffer easily e.g. `Buffer('abc').slice(0,5).length === 3`